bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/RuleStatus.java

```diff
@@ -18,7 +18,8 @@
     UNINITIALIZED(1),
     INITIALIZING(2),
     IDLE(3),
-    RUNNING(4);
+    RUNNING(4),
+    TRIGGERED(5);
```

Comment:
How about putting "TRIGGERED" before "RUNNING"? Because that is also how the lifecycle of a rule will be